Traducir.Core/Services/SOStringService.cs

```diff
@@ -596,5 +597,30 @@ Update Strings
             }
             return _strings;
         }
+
+        public async Task<bool> DeleteSuggestionAsync(SOStringSuggestion suggestion)
+        {
+            using (var db = _dbService.GetConnection())
+            {
+                await db.ExecuteAsync(@"
+Update StringSuggestions
+Set StateId = {=DeletedByOwner}
+Where Id = @suggestionId;
+
+Insert Into StringSuggestionHistory
+            (StringSuggestionId, HistoryTypeId, UserId, Comment, CreationDate)
+Values      (@suggestionID, {=DeletedByOwner}, @userId, @comment, @now);",
```

Comment:
hmm, does this actually work? I was sure Dapper was case sensitive (therefore, `@suggestionID` wouldn't match the `suggestionId` property of the parameters object you're passing)